src/forms/FormHeading.tsx

```diff
@@ -0,0 +1,27 @@
+import { Css, Margin, Xss } from "src/Css";
+
+export interface FormHeadingProps {
+  title: string;
+  xss?: Xss<Margin>;
+  // This is passed automatically by FormLines
+  isFirst?: boolean;
+}
+
+export function FormHeading(props: FormHeadingProps) {
+  const { title, xss, isFirst = false } = props;
+  return (
+    <div
```

Comment:
Since this is a "Heading" component I'm tempted to use a `hX` tag, but I also don't know which one should we use... I'd guess h3/h4, but that is kind of an odd assumption.

Reply:
Ah yeah, good call, I get kind of lazy on that; will make it an `h3`.